en/micro-integrator/docs/references/connectors/amazonsqs-connector/amazonsqs-inbound-endpoint-example.md

```diff
@@ -0,0 +1,108 @@
+
+## Configure inbound endpoint using WSO2 Integration Studio
+
+1. Download [WSO2 Integration Studio](https://wso2.com/integration/integration-studio/). Create an **Integration Project** as below. 
+   
+   <img src="../../../../assets/img/connectors/integration-project.png" title="Creating a new Integration Project" width="800" alt="Creating a new Integration Project" />
+
+2. Right click on **Created Integration Project** -> **New** -> **Inbound Endpoint** -> **Create A New Inbound Endpoint** -> **Inbound Endpoint Creation Type**and select as **custom** -> Click **Next**.
+   
+   <img src="../../../../assets/img/connectors/smpp-inboundEP-create-new-IE.png" title="Creating inbound endpoint" width="400" alt="Creating inbound endpoint" style="border:1px solid black"/>
+
+3. Click on **Inbound Endpoint** in design view and under `properties` tab, update class name to `org.wso2.carbon.inbound.amazonsqs.AmazonSQSPollingConsumer`. 
+
+4. Navigate to the source view and update it with the following configuration as required. 
+     
+   ```xml
+   <?xml version="1.0" encoding="UTF-8"?>
+   <inboundEndpoint xmlns="http://ws.apache.org/ns/synapse"   
+                    name="AmazonSQS"    
+                    onError="requestISO" 
+                    sequence="request"    
+                    suspend="false" 
+                    class="org.wso2.carbon.inbound.amazonsqs.AmazonSQSPollingConsumer">
+       <parameters>
+           <parameter name="sequential">true</parameter>
+           <parameter name="interval">2000</parameter>
+           <parameter name="coordination">true</parameter>
+           <parameter name="waitTime">19</parameter>
+           <parameter name="maxNoOfMessage">10</parameter>
+           <parameter name="destination">https://sqs.us-east-2.amazonaws.com/610968236798/eiconnectortestSQS</parameter>
+           <parameter name="accessKey">AKIAY4QELOL7GF35XBW5</parameter>
+           <parameter name="secretKey">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</parameter>
+           <parameter name="attributeNames">attributeName1,contentType</parameter>
+           <parameter name="contentType">application/json</parameter>
+           <parameter name="class">org.wso2.carbon.inbound.amazonsqs.AmazonSQSPollingConsumer</parameter>
+           <parameter name="inbound.behavior">polling</parameter>
+       </parameters>
+   </inboundEndpoint>   
+   ```   
+   **Sequence to process the message**
+   
+   In this example, for simplicity we will just log the message, but in a real world use case, this can be any type of message mediation.
+ 
+   ```xml
+   <?xml version="1.0" encoding="UTF-8"?>
+   <sequence name="request" trace="disable" xmlns="http://ws.apache.org/ns/synapse">
+       <log level="full"/>
+   </sequence>
+   ```
+> **Note**: To configure the `secretKey` and `accessKey` parameter value, please use the [Setting up the Amazon Lambda Environment](../amazonlambda-connector/setting-up-amazonlambda.md) documentation.
```

Comment:
Why Lambda? Shouldn't it be SQS?

Reply:
No.The reason is generating `secretKey` and `accessKey` part is similar to the Amazon Lambda connector.